src/frontend-samples/camera-path-sample/CameraPathWidget.tsx

```diff
@@ -170,8 +170,8 @@ const CameraPathWidget: React.FunctionComponent = () => {
   const _onChangeRenderSpeed = (currSpeed: string) => {
     let speedOfMotion: number = 0;
     switch (currSpeed) {
-      case "5 Mph: Walk":
-        speedOfMotion = 2.2352; // 4mph = 1.788 meters/second
+      case "5 Mph: Walking":
```

Comment:
The initial `setSpeedLevel` sets it to "3 Mph: Walking". Maybe these options should be an enum or something?

Reply:
refactored. Made a list that tightly couples the text and m/s together. See if that looks better